core/rpc/src/Mu/Server.hs

```diff
@@ -60,13 +86,14 @@ instance HandlesRef ('FromRegistry subject t last) t
 -- Arguments
 instance (HandlesRef ref t, Handles args ret m h, handler ~ (t -> h))
          => Handles ('ArgSingle ref ': args) ret m handler
-instance (HandlesRef ref t, Handles args ret m h, handler ~ (ConduitT () t IO () -> h))
+instance (HandlesRef ref t, Handles args ret m h, handler ~ (ConduitT () t m () -> h))
```

Comment:
Now that we're handling it with `ServerError`... Should we keep this instance? this might be a scapehatch for using monads that don't have a `MonadError ServerError`

Reply:
Indeed, this was wrong. I just pushed an update to require `m` to be a `MonadError`.